File: Android Kotlin Retrofit Node Express- Orphanslife/app/src/main/java/com/kaligotla/orphanslife/model/Sponsor.kt
package com.kaligotla.orphanslife.model

import com.google.gson.annotations.SerializedName

data class Sponsor(
    @SerializedName("sponsor_id")
    val sponsor_id: Int,

    @SerializedName("sponsor_name")
    val sponsor_name: String,

    @SerializedName("sponsor_dob")
    val sponsor_dob: String,

    @SerializedName("sponsor_gender")
    val sponsor_gender: String,

    @SerializedName("sponsor_govt_id_type")
    val sponsor_govt_id_type: String,

    @SerializedName("sponsor_govt_id")
    val sponsor_govt_id: String,

    @SerializedName("sponsor_mobile")
    val sponsor_mobile: String,

    @SerializedName("sponsor_email")
    val sponsor_email: String,

    @SerializedName("sponsor_password")
    val sponsor_password: String,

    @SerializedName("marital_status")
    val marital_status: String,

    @SerializedName("sponsor_image")
    val sponsor_image: String,

    @SerializedName("sponsor_address")
    val sponsor_address: String,

    @SerializedName("sponsor_location")
    val sponsor_location: Location,

    @SerializedName("spouce_name")
    val spouce_name: String,

    @SerializedName("spouce_dob")
    val spouce_dob: String,

    @SerializedName("spouce_govt_id_type")
    val spouce_govt_id_type: String,

    @SerializedName("spouce_govt_id")
    val spouce_govt_id: String,

    @SerializedName("spouce_mobile")
    val spouce_mobile: String,

    @SerializedName("spouce_image")
    val spouce_image: String,

    @SerializedName("donation")
    val donation: Donation,

    @SerializedName("sponsor_created_at")
    val sponsor_created_at: String,

    @SerializedName("sponsor_updated_at")
    val sponsor_updated_at: String
)

File: Flutter Dart Node Express - Orphanslife/lib/Models/Sponsor.dart
import 'dart:convert';
import 'package:json_annotation/json_annotation.dart';

@JsonSerializable()
class Sponsor {
  int sponsor_id;
  String sponsor_name, sponsor_dob, sponsor_gender, sponsor_govt_id_type,
      sponsor_govt_id, sponsor_mobile, sponsor_email, sponsor_password, marital_status,
      sponsor_image, sponsor_address;
  int sponsor_location_id;
  String spouce_name, spouce_dob, spouce_govt_id_type, spouce_govt_id, spouce_mobile,
      spouce_image;
  int donation_id;
  String created_at, updated_at;

  Sponsor.all({this.sponsor_id,this.sponsor_name,this.sponsor_dob,this.sponsor_gender,
  this.sponsor_govt_id_type,this.sponsor_govt_id,this.sponsor_mobile,this.sponsor_email,this.sponsor_password,
  this.marital_status,this.sponsor_image,this.sponsor_address,this.sponsor_location_id,
  this.spouce_name,this.spouce_dob,this.spouce_govt_id_type,this.spouce_govt_id,
  this.spouce_mobile,this.spouce_image,this.donation_id,this.created_at,this.updated_at});

  factory Sponsor.fromJson(Map<String, dynamic> json) => Sponsor.all();

  Map<String, dynamic> toJson() => {
    "sponsor_id": sponsor_id,
    "sponsor_name": sponsor_name,
    "sponsor_dob": sponsor_dob,
    "sponsor_gender" : sponsor_gender,
    "sponsor_govt_id_type" : sponsor_govt_id_type,
    "sponsor_govt_id" : sponsor_govt_id,
    "sponsor_mobile" : sponsor_mobile,
    "sponsor_email": sponsor_email,
    "sponsor_password" : sponsor_password,
    "marital_status" : marital_status,
    "sponsor_image" : sponsor_image,
    "sponsor_address": sponsor_address,
    "sponsor_location_id": sponsor_location_id,
    "spouce_name" : spouce_name,
    "spouce_dob" : spouce_dob,
    "spouce_govt_id_type" : spouce_govt_id_type,
    "spouce_govt_id" : spouce_govt_id,
    "spouce_mobile" : spouce_mobile,
    "spouce_image" : spouce_image,
    "donation_id" : donation_id,
    "created_at" : created_at,
    "updated_at" : updated_at,
  };

}